### ISSUE TITLE
UI Suggestion - indicate which repo had the the problem in admin notices

### ISSUE THREAD

**Issue by ckuethe** [go-gitea/gitea#25496]
### Description

Already searched back issues, yes, I'm just about to go search my logs and adjust my timeouts some more... but I'd find it useful to know which repository had the failure, what operation (pull, gc, etc.) failed, and what the insufficient timeout was. For the admin message below, I'd prefer to see something like this:

`Cron: Update Mirror /data/git/repository/mirrors/linux cancelled: context deadline exceeded after 3600s`

### View Notice Details
---
Jun 24, 2023, 12:33 PM PDT
`Cron: Update Mirrors cancelled: context deadline exceeded`

### Gitea Version

1.20.0+rc0, docker

### Can you reproduce the bug on the Gitea demo site?

No

### Browser Version

Chrome 114

**Comment by ckuethe** [comment]
`Failed to update mirror repository '/data/git/repositories/mirrors/...': fatal: unable to access 'https://github.com/...': Failed to connect to github.com port 443 after 129424 ms: Couldn't connect to server`

Here's an example of a very helpful log: it tells me which repo mirror failed, why it failed, the timeout (if appropriate).

**Comment by silverwind** [comment]
Yes, this is needed. I always wonder where editorconfig errors come from.
